Project EFT/Views/Shared/Problem.cshtml

```diff
@@ -19,6 +19,10 @@
             <div></div>
         </form>
     }
+    else if (session.ContainsKey("userInfo") && !((bool)ViewData["showProblem"]))
+    {
+        <br /><br /><div>Congratulations, this problem has been completed!</div>
+    }
```

Comment:
if contains key
---- if value in map is true
---- else
else

Reply:
In reference to this and the preceding code block, which got cut off in the MLC:
`@if (session.ContainsKey("userInfo") && (bool)ViewData["showProblem"])`
[this code snippet]
`else{ [not logged in as a user case] }`